src/databricks/labs/ucx/source_code/table_info.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+import logging
+from collections.abc import Sequence, Iterable
+
+from databricks.labs.ucx.framework.crawlers import CrawlerBase
+from databricks.labs.lsql.backends import SqlBackend
+from databricks.sdk.errors import DatabricksError
+
+from databricks.labs.ucx.framework.utils import escape_sql_identifier
+from databricks.labs.ucx.source_code.base import TableInfo
+
+logger = logging.getLogger(__name__)
+
+
+class TableInfoCrawler(CrawlerBase[TableInfo]):
```

Comment:
```suggestion
class TableUsageCrawler(CrawlerBase[TableInfo]):
```

also, it looks very much similar to `DirectFsAccessCrawler`, just a different class. @asnare how can we make it more generic?

Reply:
We might want to wait for the MultiCrawler to answer that question ?